### ISSUE TITLE
[1.11] Can't Craft Workbench

### ISSUE THREAD

**Issue by hennign** [sokratis12GR/ArmorPlus#82]
Can you tell me how to craft the workbench (first arp crafting table)? I looked through the code and I thought that this:

```
manager.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.arpWorkbench),
                "LCL",
                "OTO",
                "O O",
                'T', "workbench",
                'O', "blockCoal",
                'L', "gemLapis",
                'C', "gemLavaCrystal"));
```
was the recipe, and it matches what I remember from previous versions, but making that recipe doesn't do anything. I'm assuming workbench refers to the regular crafting table.

**Comment by sokratis12GR** [comment]
Alright, thanks for the report going to take a look

**Comment by sokratis12GR** [comment]
Fixed in [armorplus-1.11-9.2.0.1-beta](https://minecraft.curseforge.com/projects/armorplus/files/2359042)
